datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1467,6 +1467,14 @@ CREATE VIEW protocol_upgrade_proposals_current AS (
       FROM protocol_upgrade_proposals
   ORDER BY upgrade_block_height, vega_release_tag, vega_time DESC);
 
+CREATE TABLE IF NOT EXISTS core_snapshots(
```

Comment:
It think you're missing the `drop table if exists` bit for this somewhere at the bottom of the file.